modules/ROOT/pages/push/binding.adoc

```diff
@@ -10,52 +10,55 @@ include::{partialsdir}/attributes.adoc[]
 .Android
 
 ****
-* [x] A Google account
+* [x] Google account
 * [x] Access to the Firebase console. Refer to link:https://firebase.google.com/docs/[Firebase Docs].
 ****
 
 [role="secondary"]
 .iOS
 
 ****
-* [x] An Apple Developer account
-* [x] An APNs client TLS certificate. Refer to link:https://help.apple.com/developer-account/#/dev82a71386a[official Apple guide].
+* [x] Apple Developer account
+* [x] APNs client TLS certificate. Refer to link:https://help.apple.com/developer-account/#/dev82a71386a[official Apple guide].
 ****
 
 
 include::{partialsdir}/generic-binding.adoc[]
 
-When you bind a mobile client to the {unifiedpush-service} service, a variant is created to store the credentials required to communicate with a push network.
-Those credentials need to be provided in the binding dialog.
+=== Required Binding Parameters
 
-First pick your *Mobile Client Type*. Depending on the selection you then have to fill out the fields for either Android or iOS.
+Choose a *Mobile Client Type* and fill out the required fields:
 
 [role="primary"]
 .Android
 
 ****
-You need to provide a *Server Key* and a *Sender ID*. Both can be found in the Firebase console under *Project Settings > Cloud Messaging*. Make sure that you have selected `Android` in the `Mobile Client Type` dropdown.
+- Enter a *Server Key* and a *Sender ID*. Both are present in the Firebase console.
+- Ensure that you have selected `Android` in the `Mobile Client Type` dropdown menu.
 ****
 
 [role="secondary"]
 .iOS
 
 ****
-You need an APNs client TLS certificate. Have a look at link:apple-setup.html[APNs Setup] for instructions on how to obtain one. Make sure that you have selected `iOS` in the `Mobile Client Type` dropdown.
-The certificate needs to be encoded in base64 before copying it into the field labelled `iOS .p12 file`. If you're on Mac OS run:
-
+- Ensure that you have selected `iOS` in the `Mobile Client Type` dropdown menu.
+- Obtain an APNs client TLS certificate. Refer to link:https://help.apple.com/developer-account/#/dev82a71386a[official Apple guide]. 
+- Encode the certificate using base64 and paste the encoded certificate into the field labeled `iOS .p12 file`. 
++
+If you are using macOS, run:
++
 ----
 cat <path/to/your/.p12> | base64
 ----
-
-If you're on Linux chances are that you're using a version of the base64 tool that does line wrapping. You need to disable this:
-
++
+If you are using Linux, chances are that you are using a version of the base64 tool that does line wrapping.
+Disable base64 line wrapping by using `--wrap=0` flag:
++
 ----
 cat <path/to/your/.p12> | base64 --wrap=0
 ----
 
 ****
-
-After clicking Bind your variant will be created in UPS and you should see a new binding for your mobile client.
-
-
+[start=6]
+. Click *Bind*.
+Unified Push server creates a new variant and the OpenShift console displays a new binding for your mobile client. A variant is used for the storage of the credentials provided in the Binding procedure.
```

Comment:
UPS? followed up in mail

Reply:
i'm leaving this as is as per email for now

Reply:
+1